service/src/main/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelper.java

```diff
@@ -27,6 +35,25 @@ public static String buildSubjectLine(final CaseData caseData) {
             .collect(joining(", "));
     }
 
+    public static String buildSubjectLineWithHearingBookingDateSuffix(final String subjectLine,
+                                                                      final List<Element<HearingBooking>>
+                                                                          hearingBookings) {
+        String hearingDateText = "";
+        if (isNotEmpty(hearingBookings)) {
+            hearingDateText = buildHearingDateText(hearingBookings);
+        }
+
+        return Stream.of(subjectLine, hearingDateText)
+            .filter(StringUtils::isNotBlank)
+            .collect(joining(","));
```

Comment:
should it be ", " just like in `buildSubjectLine`?

Reply:
yes, as `buildHearingDateText(...)` adds the needed space